### ISSUE TITLE
Lieutenant General Andorov is spawned by default.

### ISSUE THREAD

**Issue by Cabro** [elysium-project/server#3169]
http://wowwiki.wikia.com/wiki/Lieutenant_General_Andorov

This NPC, along with the 4 guards, should spawn after Kurinaxx is defeated.

**Comment by BrynjaDarrow** [comment]
Yeah, it looks really weird if they just stand around Kurinaxx doing nothing. Definitely breaks immersion.
